client/src/utils/apiWrapper.ts

```diff
@@ -133,10 +133,10 @@ export const getUsers = (): Promise<
  * Returns all unclaimed and approved pitches
  * Returns GET_UNCLAIMED_PITCHES_FAIL upon failure
  */
-export const getUnclaimedPitches = (): Promise<
+export const getPitches = (): Promise<
   AxiosResponse<GetPitchesResponseType> | ErrorWrapper
 > => {
-  const requestString = `${BASE_URL}/pitch?unclaimed=true`;
+  const requestString = `${BASE_URL}/pitch`;
```

Comment:
I think you might have to modify the pitch.ts file because this would get all the pitches, even the ones that haven't been approved by the staff yet.